scripts/demo.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-# Quick-start: run this
+# Quick-start: run this from the base jiant/ directory
```

Comment:
Optional, but I like to add a `pushd` to automatically jump to this directory - just smooths out the experience a bit and avoids annoying path errors.

Something like:
```
pushd $(dirname $0)
pushd $(git rev-parse --show-toplevel)
```